src/psm/src/ir_solver.cpp

```diff
@@ -731,9 +731,14 @@ void IRSolver::createGmatWireNodes(const vector<dbSBox*>& power_wires,
       }
       length = v_itr - v_prev;
       if (length > node_density_) {
-        for (int v_i = v_prev+ node_density_; v_i < v_itr;
-               v_i += node_density_) {
-          Gmat_->setNode({v_i, y_loc1}, l); //assumes bottom layer is always horizontal
+        int num_nodes = length / node_density_; //truncated integer will always be>1
+        if(length % node_density_ == 0){
+          num_nodes -=1;
+        }
```

Comment:
if length = 1.1* node_density_ then num_nodes = 1 and is not >1.  Did you mean >= 1 ?